app/scripts/controllers/loanAccount/NewLoanAccAppController.js

```diff
@@ -265,6 +265,7 @@
                     scope.formData.collateral = [];
                     for (var i in scope.collaterals) {
                         scope.formData.collateral.push({type: scope.collaterals[i].type, value: scope.collaterals[i].value, description: scope.collaterals[i].description});
+                        
```

Comment:
@MYKatz  Please remove this extra line, you can remove the commit. It is not needed. Hint: `do git rebase -i HEAD~2` and delete this commit by prefixing it with `d`. Ping me if you have any doubts